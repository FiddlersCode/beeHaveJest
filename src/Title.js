import React, { Component } from 'react';
import { View, Text, StyleSheet } from 'react-native';


const Title = () => {
  return (
    <View style={styles.container}>
      <Text style={styles.welcome}>BeeHappy</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  welcome: {
    fontSize: 20,
    textAlign: 'center',
    margin: 10,
  }
});
export default Title;
