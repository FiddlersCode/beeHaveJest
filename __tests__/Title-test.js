'use strict';

import 'react-native'
import React from 'react'
import Title from '../src/Title.js'
import renderer from 'react-test-renderer'

test('renders correctly', () =>{
  const tree = renderer.create (
    <Title />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
