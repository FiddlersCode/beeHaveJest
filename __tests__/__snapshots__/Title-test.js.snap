// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    BeeHappy
  </Text>
</View>
`;
